spec/rubocop/cop/lint/unused_method_argument_spec.rb

```diff
@@ -70,17 +70,6 @@ def some_method(foo, bar)
       end
     end
 
-    context 'when a required keyword argument is unused', ruby: 2.1 do
```

Comment:
Here's a late comment. Keyword arguments without defaults were introduced in Ruby 2.1, so I think this example is for Ruby >= 2.1, rather than Ruby < 2.2, i.e., it should be kept.

Reply:
Sure. I opened a PR #6104.